src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -45,6 +48,7 @@ class FrontendGenImpl {
   }
 
   mlir::ModuleOp ImportONNXModel(const onnx::ModelProto &model) {
+    SetOpSetImport(model); // Determines which opsets to use.
```

Comment:
Could you give us a few more details on what this does and how it particularly helps with handling function ops?

Reply:
I added comments to explain this. It essentially maintains a map from each op-set domain (like the standard ONNX domain) to the version of that domain (e.g., 11 or 12) used in the current model. It should be used to resolve any operator name (including function name).